docker-compose.yml

```diff
@@ -32,14 +32,17 @@ services:
     environment:
       PGUSER: postgres
       PGPASSWORD: postgres
-    command: 
+    volumes:
+      - ./testdata:/testdata
+    command:
       - /bin/bash
       - -c
       - |
         wget "https://github.com/go-spatial/tegola-testdata/raw/master/tegola.dump" 
```

Comment:
Note, I'm fine adding this file to our repo. It's only 3.5mb. Younger me cared about not having binaries in the repo, but this is test data and I think it should live next to the source code that uses it.

Reply:
Do you want me to pull that pg dump into tegola?

Reply:
I'm pulling the `tegola.dump` into the tegola repository.